src/ucs/debug/debug.c

```diff
@@ -531,12 +531,12 @@ ucs_status_t ucs_debug_lookup_address(void *address, ucs_debug_address_info_t *i
         return UCS_ERR_NO_ELEM;
     }
 
-    strncpy(info->file.path, dlinfo.dli_fname, sizeof(info->file.path));
+    ucs_safe_strncpy(info->file.path, dlinfo.dli_fname, sizeof(info->file.path));
```

Comment:
the right solution IMHO is to pass `sizeof(info->file.path)-1` instead of `sizeof(info->file.path)`

Reply:
in this case it may cut string (and doesn't resolve compilation error)

Reply:
it should be cut the string anyway if it's too long
maybe add better implementation of ucs_safe_strncpy() which also makes sure the destination would be NULL-terminated?

Reply:
done